planetiler-core/src/main/java/com/onthegomap/planetiler/util/LoopLineMerger.java

```diff
@@ -360,6 +419,24 @@ public void remove() {
       to.removeEdge(reversed);
     }
 
+    double angleTo(Edge other) {
+      if (!from.equals(other.from)) {
+        // Todo raise some error, the edges don't start at same coordinate
+        System.out.println("error start point mismatch");
+        return 0.0;
+      }
+      if (coordinates.size() < 2 || other.coordinates.size() < 2) {
+        // todo no real linestring...
+        System.out.println("error lines are points");
+        return 0.0;
+      }
```

Comment:
you could probably just do `assert from.equals(other.from)` and `assert coordinates.size() >= 2` here too just to make sure they hold when we're running unit tests but not introduce any runtime overhead.

Reply:
OK